README.md

```diff
@@ -1,101 +1,106 @@
 # Kameleoon AWS Lambda@Edge Starter Kit
 
-> Kameleoon AWS Lambda@Edge Starter Kit to run Kameleoon Experimentation and Feature Flags on [Lambda@Edge](https://aws.amazon.com/lambda/edge).
+> The Kameleoon AWS Lambda@Edge Starter Kit demonstrates Kameleoon experimentation and feature flags on Amazon Web Services (AWS) [Lambda@Edge](https://aws.amazon.com/lambda/edge).
 
-This repository is home to the AWS Lambda@Edge Starter Kit, providing a quickstart for users who want to use [Kameleoon NodeJS SDK](https://developers.kameleoon.com/feature-management-and-experimentation/web-sdks/nodejs-sdk) with AWS Lambda@Edge. Kameleoon is a powerful experimentation and personalization platform for product teams, enabling insightful discoveries through every feature on your roadmap. Discover more at https://kameleoon.com, or see the [developer documentation](https://developers.kameleoon.com).
+This repository is home to the Kameleoon starter kit for AWS Lambda@Edge. Kameleoon is a powerful experimentation and personalization platform for product teams that enables you to make insightful discoveries by testing the features on your roadmap. Discover more at https://kameleoon.com, or see the [developer documentation](https://developers.kameleoon.com).
 
-## Getting Started
+## Getting started
 
-### Prerequisites
+This starter kit provides quickstart instructions for developers using the [Kameleoon NodeJS SDK](https://developers.kameleoon.com/feature-management-and-experimentation/web-sdks/nodejs-sdk) with AWS Lambda@Edge.
 
-These steps should be completed before you get started:
+### Prerequisites
 
-1. You need a **Kameleoon Account**.
+Make sure you have the following requirements before you get started:
 
-2. You need an AWS Account with Lambda@Edge Access. For more information, visit the official [AWS Lambda@Edge product page here](https://aws.amazon.com/lambda/edge).
+1. A Kameleoon user account. Visit [kameleoon.com](https://www.kameleoon.com/) to learn more.
+1. The [Kameleoon NodeJS SDK](https://developers.kameleoon.com/feature-management-and-experimentation/web-sdks/js-sdk) installed with some feature flags or experiments already configured.
```

Comment:
incorrect link, it redirects to JS SDK instead of NodeJS SDK

Reply:
Good catch! Thanks!